srcpkgs/pinebookpro-uboot/template

```diff
@@ -2,9 +2,8 @@
 pkgname=pinebookpro-uboot
 reverts="20200212_1 20200212_2"
 version=0.0.20200212
-revision=3
+revision=4
 _commit_uboot=365495a329c8e92ca4c134562d091df71b75845e
-_commit_atf=22d12c4148c373932a7a81e5d1c59a767e143ac2
 archs="aarch64*"
 hostmakedepends="git flex cross-arm-none-eabi dtc python3 bc"
```

Comment:
You have to include the new atf package in makedepends.

Reply:
ah, right. My bad.